examples/basic_example.py

```diff
@@ -63,11 +63,11 @@ def future_loading(t, x = None):
     print('\tP(Success) if mission ends at 3002.25: ', metrics.prob_success(eol, 3005.25))
 
     # Plot state transition 
-    fig = states.snapshot(0).plot_scatter(label = "t={}".format(int(times[0][0])))
-    states.snapshot(10).plot_scatter(fig = fig, label = "t={}".format(int(times[0][10])))
-    states.snapshot(50).plot_scatter(fig = fig, label = "t={}".format(int(times[0][50])))
+    fig = states.snapshot(0).plot_scatter(label = "t={}".format(int(times[0])))
```

Comment:
times returned from prediction is now times[time_index] instead of times[sample_id][time_index]. This is a breaking change that requires users to update how they're using it (like I do here).